pkg/policy/policiesSchema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "enum": [
+        "v1"
+      ]
+    },
+    "message": {
+      "type": "string",
+      "enum": [
+        "Should have exact one policy set as default",
```

Comment:
where are the checks for these 3?
"Should have exact one policy set as default"
"All policy names should be unique"
"All rule identifiers should be unique"